docs/scripts.rst

```diff
@@ -0,0 +1,74 @@
+***********************************
```

Comment:
I wonder if this is "allowed".
AFAIK these lines should have the length of the word(s) they are enclosing.

Reply:
I would agree.

Reply:
Fixed by force push. Thanks!